music21/stream/__init__.py

```diff
@@ -8936,6 +8930,9 @@ def findConsecutiveNotes(self,
 
         for v in vGroups:
             for e in v:
+                # Filter out all but notes and rests
+                if not isinstance(e, note.GeneralNote):
+                    continue
```

Comment:
Thank, you.  This is great.

btw -- we use `if 'GeneralNote' not in e.classes:` or `not in e.classSet` a lot in the code, because it's actually an order of magnitude faster than isinstance (or was 5 years ago last time I did a lot of speed profiling).  findConsecutiveNotes isn't a performance critical routine, so no-big-deal, but something to know for the future.